src/build/languageServerManager.ts

```diff
@@ -35,12 +36,10 @@ export class LanguageServerManager {
         return this._languageServerStatus;
     }
 
-    private readyToStartLanguageServer(): boolean {
-        return this._environmentController.enableAutomaticRealTimeValidation && this._languageServerStatus === 'Idle';
-    }
-
-    private tryToStartLanguageServer(): void {
-        this._languageServerStatus = 'Starting';
-        vscode.commands.executeCommand('docs.enableRealTimeValidation');
+    public tryToStartLanguageServer(): void {
```

Comment:
my fault, better to use `startLanguageServer` directly, `tryTo` means it may fails and need to handle